src/octopeer-github/main/Options/Options.ts

```diff
@@ -25,14 +25,15 @@ const Options = new (class Options {
     public get DATA_TABS() {        return "dataTabs"; }
     public get DATA_RESOLUTION() {  return "dataResolution"; }
 
-    /**
-     * A option on the left should be disabled if the option on the right should be disabled.
-     */
-    public optionDependencies: {[name: string]: string; } = {
-        [this.DATA_HTML]:       this.DATA_KEYSTROKES,
-        [this.DATA_KEYSTROKES]: this.DATA_COMMENTS ,
-        [this.MOUSE_CLICK]:     this.MOUSE_HOVER,
-        [this.MOUSE_HOVER]:     this.MOUSE_POSITION,
+    // A option on the left should be disabled if the option on the right should be disabled.
+    public get optionDependencies(): {[name: string]: string; }{
```

Comment:
Because this is a constant now, please rename `optionsDependencies` to `DEPENDENCIES` so you can access it with `Options.DEPENDENCIES`.